projects/igniteui-angular/src/lib/progressbar/linearbar.component.spec.ts

```diff
@@ -1,302 +1,185 @@
 import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
-import { IgxLinearProgressBarComponent, toPercent } from './progressbar.component';
-
+import { IgxLinearProgressBarComponent } from './progressbar.component';
 import { configureTestSuite } from '../test-utils/configure-suite';
 
-const SUCCESS_TYPE_CLASS = 'igx-linear-bar--success';
-const INFO_TYPE_CLASS = 'igx-linear-bar--info';
-const STRIPED_CLASS = 'igx-linear-bar--striped';
-const LINEAR_BAR_TAG = 'igx-linear-bar';
-const INDETERMINATE_CLASS = 'igx-linear-bar--indeterminate';
-
-describe('IgxLinearBar', () => {
-    let progress: IgxLinearProgressBarComponent;
+fdescribe('IgxLinearProgressBarComponent', () => {
     let fixture: ComponentFixture<IgxLinearProgressBarComponent>;
+    let progress: IgxLinearProgressBarComponent;
+    let linearBar: HTMLElement;
 
     configureTestSuite();
 
     beforeAll(waitForAsync(() => {
         TestBed.configureTestingModule({
-    imports: [IgxLinearProgressBarComponent]
-})
-        .compileComponents();
+            imports: [IgxLinearProgressBarComponent]
+        }).compileComponents();
     }));
 
     beforeEach(() => {
+        TestBed.configureTestingModule({
+            imports: [IgxLinearProgressBarComponent]
+        }).compileComponents();
+
         fixture = TestBed.createComponent(IgxLinearProgressBarComponent);
         progress = fixture.componentInstance;
-        progress.animate = false;
         fixture.detectChanges();
+        linearBar = fixture.debugElement.nativeElement;
     });
 
-    it('should initialize linearProgressbar with default values', () => {
-        const domProgress = fixture.debugElement.nativeElement;
-        const defaultMaxValue = 100;
-        const defaultValue = 0;
-        const defaultStriped = false;
-        const defaultType = 'default';
+    function clasListContains(element: HTMLElement, className: string, expected: boolean) {
+        expect(element.classList.contains(className)).toBe(expected);
+    }
```

Comment:
```suggestion
    function hasClass(element: HTMLElement, className: string, expected: boolean) {
        expect(element.classList.contains(className)).toBe(expected);
    }
```

This function is declared both here and in the circular tests. Extract it into the helper-utils.spec.ts in the test-utils folder.